scss/variables/typography.js

```diff
@@ -52,7 +52,7 @@ chassis.typography = {
 	},
 	"sans": {
 		name: "Font Family - Sans",
-		value: "Helvetica, Arial, sans-serif"
+		value: "Open Sans, Helvetica, Arial, sans-serif"
```

Comment:
Open Sans can be arguably better than helvetica, but something that should be discussed in meeting